CHANGELOG.md

```diff
@@ -8,6 +8,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 - Disallow incorrect comparison of atomic values in a non-atomic way.
 
+### Removed
+- Remove dependency on `golang.org/x/{lint, tools}`.
```

Comment:
or maybe just `Moved out test dependencies` (since it doesn't affect module resolution etc, just `go mod download`)

Reply:
We're not dropping all test dependencies, though. Only dev-time dependencies.